src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  // The car coste calculator
+  let result = days * 40;
+
+  if (days >= '7') {
+    result -= '50';
+
+    return result;
```

Comment:
you can change result to const and return the outcome of operation directly
also then change naming of result to be more descriptive ;)